src/types/const.ts

```diff
@@ -22,8 +22,12 @@ export enum SelectionType {
 
 export const CL_IMPORT_ID = '9c110735ea8b440d8f31c5c68ffb767d'
 
+export const LOCATION_HOST : Readonly<string> = location.hostname
```

Comment:
Hmm, is this planned to change in future and prevent the changes in dependent files?

Looks like just a constant mapped to other constant, was wondering if we could just put `location.hostname` in place of `LOCATION_HOSTNAME`.  My first thought if I saw `LOCATION_HOSTNAME` is "Wonder why this is here? Why aren't they using the browser's location object, must be something special" Then I see it is using `location.hostname` just hidden.

Reply:
The idea was making this configurable later. If UI wants to be served by a server that is different than the host that serves the bot, i.e. SDK and directline offline. probably better name is BOT_HOSTNAME instead of LOCATION_HOSTNAME. What do you think?

Reply:
Oh ok. Yea that name, bot hostname, is better.  

I'm thinking this could be another global setting similar to the existing bot port.
My understanding is this would allow running UI locally to target a bot that is deployed or really any difference as you said because it decouples the two further.